html/index.html

```diff
@@ -104,306 +104,14 @@ <h5>LET'S CONNECT</h5>
-
-                <a href="#"><img src="../img/jumbo1.jpg" alt="Tonic_img"></a> 
-
-            </figure>
-
-            <figure class="figure2">
-
-                <a href="#"><img src="../img/jumbo5.png" alt="Nature_img"></a>
-
-            </figure>
-
-            <div class="article1div show1">
-
-                <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
-
-                <ul>
-
-                    <li><a class="textdecoration font2" href="#">CANOPY</a></li>
-                    <li><a class="textdecoration" href="#"><img src="../img/mid_dot.jpg" alt="mit_dot"></a></li>
-                    <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
-                    <li class="textdecoration"><a href="#"><img src="../img/mid_dot.jpg" alt="mit_dot"></a></li>
-                    <li><a class="textdecoration font3" href="#">2015</a></li>
-
-
-                </ul>
-
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-
-                <ul class="gridbox1">
-
-                    <li class="background1"><a class="textdecoration font4" href="#">html</a></li>
-                    <li class="background1"><a class="textdecoration font4" href="#">css</a></li>
-                    <li class="background1"><a class="textdecoration font4" href="#">javaScript</a></li>
-
-                </ul>
-
-                <button type="submit"><a class="font5 textdecoration" href="#">See Project</a></button>
-
-            </div>
-
-            <div class="article1div show2">
-
-                <h3><a class="textdecoration font1" href="#">Tonic</a></h3>
-
-                <ul class="ul1">
-
-                    <li><a class="textdecoration font2" href="#">CANOPY</a></li>
-                    <li><a class="textdecoration" href="#"><img src="../img/mid_dot.jpg" alt=""></a></li>
-                    <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
-                    <li><a class="textdecoration" href="#"><img src="../img/mid_dot.jpg" alt=""></a></li>
-                    <li><a class="textdecoration font3" href="#">2015</a></li>
-
-
-                </ul>
-
-                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-
-                <ul class="gridbox1">
-
-                    <li class="background1"><a class="textdecoration font4" href="#">html</a></li>
-                    <li class="background1"><a class="textdecoration font4" href="#">Ruby on rails</a></li>
-                    <li class="background1"><a class="textdecoration font4" href="#">css</a></li>
-                    <li class="background1"><a class="textdecoration font4" href="#">javaScript</a></li>
-
-                </ul>
-
-                <button type="submit"><a class="font5 textdecoration" href="#">See Project</a></button>
-
-            </div>
-
-        </article>
-
-        <article class="article2 bg_color7">
```

Comment:
- [OPTIONAL ] Kindly make  your background  couleur transparent `background: #C1C7D0;` as shown in the Figma design and it applies the all content of the section ( it takes the max screen at 100% ) as shown in your design your background color is just applied in the small part not a max screen please fix that to make your design stick to Figma :+1: :+1: 

## Your design 

![image](https://user-images.githubusercontent.com/95297251/209157032-26d96ad7-99be-4413-88dd-f213eae721b8.png)

## Figma design 

![image](https://user-images.githubusercontent.com/95297251/209160293-b6ba1583-7915-40e5-8586-0047f6168a47.png)